src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -5379,10 +5379,10 @@ public void Directive_GloballyEnabled_GeneratedCode_PartialClasses2()
 // <autogenerated />
 partial class Program
 {
-#pragma warning enable nullable
     static void G(object x)
     {
         x = null; // no warning
+#nullable enable warnings
```

Comment:
why change the location of the directive?

Reply:
This was a bug. Addressed in 71678bd.